map/main.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import "fmt"
+
+func main() {
+
+	// make(map[key type]value type)
+	m := make(map[string]int)
```

Comment:
Personally I very rarely use `make` (I only use it for slices when I know the size).
Here's an alternative way of writing this
```suggestion
	m := map[string]int{}
```